{{cookiecutter.project_slug}}/clients/web/vue3/src/services/users/api.js

```diff
@@ -8,53 +8,45 @@ import {
   loginShape,
 } from './models'
 
-const login = createCustomServiceCall(
-  {
-    inputShape: loginShape,
-    outputShape: userShape,
-  },
-  async ({ client, input, utils }) => {
+const login = createCustomServiceCall({
+  inputShape: loginShape,
+  outputShape: userShape,
+  cb: async ({ client, input, utils }) => {
     const res = await client.post('/login/', utils.toApi(input))
     return utils.fromApi(res.data)
   },
-)
-
-const requestPasswordReset = createCustomServiceCall(
-  {
-    inputShape: forgotPasswordShape,
-  },
-  async ({ client, input, utils }) => {
-    const res = await client.post('/password/reset/', utils.toApi(input))
-    return utils.fromApi(res.data)
-  },
-)
+})
 
-const resetPassword = createCustomServiceCall(
-  {
-    inputShape: resetPasswordShape,
-    outputShape: userShape,
+const requestPasswordReset = createCustomServiceCall({
+  inputShape: forgotPasswordShape,
+  cb: async ({ client, input }) => {
+    await client.post('/password/reset/', utils.toApi(input))
+    return
   },
-  async ({ client, input, utils }) => {
+})
+const resetPassword = createCustomServiceCall({
+  inputShape: { uid: z.string().email(), token: z.string(), password: z.string() },
+  outputShape: userShape,
+  cb: async ({ client, input, utils }) => {
```

Comment:
linter error is referring to `z` not being imported anymore because a prior PR changed this to be `resetPasswordShape` for the `inputShape`